src/internal-server/app/Controllers/Http/UserController.js

```diff
@@ -347,12 +334,8 @@ class UserController {
     // store the old serviceId as the key for future lookup
     serviceIdTranslation[service.serviceId] = newServiceId;
 
-    if (
-      service.settings &&
-      typeof service.settings === 'string' &&
-      service.settings.length > 0
```

Comment:
And here (?)

Reply:
that's already handled in the `convertJSON` method